src/js/modes/classic.js

```diff
@@ -0,0 +1,34 @@
+export default class Classic {
+  // name = 'classic'
+
+  constructor(app) {
+    this.app = app
+  }
+
+  init() {
+    this.app.wallet.enable()
+  }
+
+  destroy() {
+    this.app.wallet.disable()
+  }
+
+  onCoinThrow(coins) {
+    this.app.wallet.onCoinThrow(coins)
+    this.updateTitle()
+  }
+
+  onSpace(e) {
+    this.app.blocks.active.onSpace(e)
+  }
+
+  onEnter(e) {
+    this.app.blocks.active.onEnter(e)
+  }
+
+  updateTitle() {
```

Comment:
Instead of updating `document.title` here, the function should return a title string, and another part of the app would handle the DOM update.